packages/frontend/components/atomic/Dropdown/Dropdown.styles.ts

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
+
+export const Wrapper = styled.div`
+  display: inline-block;
+  position: relative;
+
+  &:focus {
+    outline: 0;
+  }
+`;
+
+export const List = styled.ul<ListProps>`
+  --dropdown-list-item-padding: ${pxToRem(8)} ${pxToRem(24)};
+
+  position: absolute;
+  top: 0;
+  left: 100%;
+  margin-inline-start: ${pxToRem(8)};
+  display: flex;
```

Comment:
nit: I don't think we need flex here either, since the dropdown items stack on each other.